src/httpAuthScheme.ts

```diff
@@ -358,30 +362,12 @@ export class Token {
         const authenticator = new Uint8Array(input.buffer.slice(offset, offset + len));
         offset += len;
 
-        return new Token(tokenTypeEntry, payload, authenticator);
-    }
-
-    static async fetch(pt: PrivateToken): Promise<Token> {
```

Comment:
given you expect this to be defined
```suggestion
abstract fetch(pt: PrivateToken): Promise<boolean>
```